src/Sulu/Component/Rest/ListBuilder/Metadata/Resources/schema/list-2.0.xsd

```diff
@@ -72,18 +72,20 @@
         <xs:attributeGroup ref="fieldAttributeGroup"/>
     </xs:complexType>
 
-    <xs:complexType name="parametersType">
+    <xs:complexType name="filterType">
         <xs:sequence>
-            <xs:element type="parameterType" name="parameter" minOccurs="1" maxOccurs="unbounded"/>
+            <xs:element type="paramType" name="param" minOccurs="0" maxOccurs="unbounded"/>
         </xs:sequence>
+        <xs:attributeGroup ref="filterAttributeGroup"/>
     </xs:complexType>
 
-    <xs:complexType name="parameterType">
-        <xs:simpleContent>
-            <xs:extension base="xs:string">
-                <xs:attribute name="key" type="xs:string"/>
-            </xs:extension>
-        </xs:simpleContent>
+    <xs:complexType name="paramType">
+        <xs:sequence>
+            <xs:element type="paramType" name="param" minOccurs="0" maxOccurs="unbounded"/>
+        </xs:sequence>
+        <xs:attribute name="name" type="xs:string"/>
+        <xs:attribute name="type" type="xs:string"/>
```

Comment:
in the `properties-1.0.xsd` we restricted this to a set of strings and set a default value